src/main/java/sirius/biz/storage/layer3/VirtualFileSystem.java

```diff
@@ -120,4 +124,37 @@ public String makePath(String... parts) {
         // checks or cleanups...
         return "/" + Strings.join("/", parts);
     }
+
+    /**
+     * Returns the storage space associated with the given file.
+     *
+     * @param virtualFile the virtual file to obtain the storage space
+     * @return the {@link BasicBlobStorageSpace} sitting on top of the virtual file
+     */
+    public Optional<? extends BasicBlobStorageSpace<?, ?, ?>> fetchBlobStorageSpace(VirtualFile virtualFile) {
+        Optional<? extends BasicBlobStorageSpace<?, ?, ?>> blobStorageSpace =
+                virtualFile.tryAs(MongoBlobStorageSpace.class);
+
+        if (blobStorageSpace.isEmpty()) {
+            blobStorageSpace = virtualFile.tryAs(SQLBlobStorageSpace.class);
+        }
+
+        return blobStorageSpace;
+    }
+
+    /**
+     * Checks if the file belongs to a work directory, dictated by storage spaces with retention days greater than zero.
+     *
+     * @param virtualFile the virtual file to check
+     * @return <tt>true</tt> when the owner space specifies retention days greater than zero, <tt>false</tt> otherwise
+     */
+    public boolean isWorkFile(VirtualFile virtualFile) {
```

Comment:
I'd use another name here, as this might imply just "/work". Maybe isTransientFile or isAutoCleanupBlob or the like.

Reply:
The name was your suggestion in the previous PR. Anyways, updated to isAutoCleanupBlob.